scripts/scil_volume_screenshot.py

```diff
@@ -0,0 +1,239 @@
+  --volume_cmap_name plasma
+
+python scil_screenshot_volume.py \
+  t1.nii.gz \
+  t1_mask_overlay.png \
+  --slices 30 40 50 60 70 80 90 100
+  --overlays brain_mask.nii.gz
+
+python scil_screenshot_volume.py \
+  t1.nii.gz \
+  t1_mask_contour.png \
+  --slices 30 40 50 60 70 80 90 100
+  --overlays brain_mask.nii.gz
+  --overlays_as_contours
+
+python scil_screenshot_volume.py \
+  t1.nii.gz \
+  t1_axial_tissue_map.png \
+  --slices 30 40 50 60 70 80 90 100 \
+  --labelmap tissue_map.nii.gz
+
+python scil_screenshot_volume.py \
+  t1.nii.gz \
+  t1_axial_tissue_viridis_cmap.png \
+  --slices 30 40 50 60 70 80 90 100 \
+  --labelmap tissue_map.nii.gz \
+  --labelmap_cmap_name viridis
+
+python scil_screenshot_volume.py \
+  t1.nii.gz \
+  t1_axial_peaks.png \
+  --slices 30 40 50 60 70 80 90 100 \
+  --peaks peaks.nii.gz \
+  --volume_opacity 0.5
+"""
+
+import argparse
+import logging
+
+from functools import partial
+from itertools import zip_longest
+import itertools
+import numpy as np
+from os.path import splitext
+
+from scilpy.io.image import assert_same_resolution
+from scilpy.io.utils import (add_default_screenshot_args,
+                             add_labelmap_screenshot_args,
+                             add_overlays_screenshot_args,
+                             add_peaks_screenshot_args,
+                             add_verbose_arg,
+                             assert_inputs_exist,
+                             assert_overlay_colors,
+                             get_default_screenshotting_data)
+from scilpy.image.utils import check_slice_indices
+from scilpy.utils.util import get_axis_index
+from scilpy.viz.screenshot import (compose_image,
+                                   screenshot_contour,
+                                   screenshot_peaks,
+                                   screenshot_volume)
+
+
+def _build_arg_parser():
+
+    p = argparse.ArgumentParser(description=__doc__,
+                                formatter_class=argparse.RawTextHelpFormatter)
+
+    sg = p.add_argument_group(title="Slicing")
+    xg = p.add_argument_group(title="Input overlays")
+    vg = p.add_argument_group(title="Volume rendering")
+    pg = p.add_argument_group(title="Peaks rendering")
+    og = p.add_argument_group(title="Overlay rendering")
+    ag = p.add_argument_group(title="Annotations")
+
+    add_default_screenshot_args(p, False, False, False, sg, ag, vg, vg)
+    add_labelmap_screenshot_args(xg, "viridis", 0.5, vg, vg)
+    add_overlays_screenshot_args(xg, 0.5, og)
+    add_peaks_screenshot_args(xg, rendering_parsing_group=pg)
+    add_verbose_arg(p)
+
+    return p
+
+
+def _parse_args(parser):
+
+    args = parser.parse_args()
+    inputs = [args.volume]
+
+    if args.overlays:
+        inputs.extend(args.overlays)
+    if args.labelmap:
+        inputs.append(args.labelmap)
+    if args.peaks:
+        inputs.extend(args.peaks)
+
+    assert_inputs_exist(parser, inputs)
+    assert_same_resolution(inputs)
```

Comment:
We now have a assert_headers_compatible. More complete than assert_same_resolution.

Also, in assert_inputs_exist, you already have the required an optional args. So:
`assert_inputs_exist(parser, args.volume, [args.overlays, args.labelmap, args.peaks]`